config/config-api/src/com/thoughtworks/go/util/command/EnvironmentVariableContext.java

```diff
@@ -227,25 +224,21 @@ public int hashCode() {
         return properties != null ? properties.hashCode() : 0;
     }
 
-    public String reportText() {
-        throw new RuntimeException("IMPLEMENT ME");
-    }
-
-    public void setupRuntimeEnvironment(Map<String, String> env, ConsoleOutputStreamConsumer consumer) {
+    public List<String> report() {
```

Comment:
@arvindsv  Pre-filled variables are system environment variables of agent process. So the behavior difference would just be saying "set 'PATH' to '/tmp/bin'" instead of "override 'PATH' with '/tmp/bin'" in the console, when user trying to override a system environment variable such as 'PATH', 'HOME'. Do you think that's worth paying extra code? If you do I can load the system envs from ProcessManager while reporting.